CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ This release has an [MSRV] of 1.75.
 
 - `Generation` on `PlainEditor` to help implement lazy drawing. ([#143] by [@xorgy])
 
+### Changed
+
+### Parley
+
+- Breaking change: `PlainEditor`'s semantics are no longer transactional ([#192][] by [@DJMcNab][])
```

Comment:
We may also want to say support for IME preedit composing text is is introduced into `PlainEditor`.